src/microbe_stage/Spawners.cs

```diff
@@ -228,6 +312,8 @@ public static void SpawnCellBurstEffect(IWorldSimulation worldSimulation, Vector
             ApplyVisualScale = Math.Abs(chunkType.ChunkScale - 1) > MathUtils.EPSILON,
         });
 
+        entity.Set(chunkType);
```

Comment:
ChunkConfiguration is not meant to be used as a component.

Reply:
How do I change chunk configuration?

Reply:
Just don't set it as a component? If you need some specific data to the passed to the chunk then either an existing component or a new component is needed. AFAIK ECS systems handle rarely accessed components pretty fine, so this could set like a component `SiderophoneHitTarget` or something that purely marks stuff siderophore can hit and has the needed info to process the hit then.

Reply:
Sorry, I don't get it. How do I change chunk configuration without .Set()ting it? Also I'm kinda lazy to do the SiderophoreHitTarget thing as chunk configuration worked pretty fine for me

Reply:
ChunkConfiguration is not meant to be used as a component type, so it should not be set as a component. It would cause all kinds of havoc with the code architecture if ChunkConfiguration was made into a component. So that's why I gave two choices: a new component that stores the needed info, or extending an existing component with the required information.